src/main/kotlin/me/misik/api/api/response/ReviewStyleResponse.kt

```diff
@@ -0,0 +1,17 @@
+package me.misik.api.api.response
+
+import me.misik.api.domain.ReviewStyle
+
+data class ReviewStyleResponse(
+    val icon: String,
+    val style: String
+) {
+    companion object {
+        fun from(reviewStyle: ReviewStyle): ReviewStyleResponse {
+            return ReviewStyleResponse(
+                icon = reviewStyle.iconUrl,
+                style = reviewStyle.name
```

Comment:
trailing comma 해주실 수 있나요?
나중에 ReviewStyleResponse 파라미터가 추가되는등의 코드변경이 발생했을때 변경내용이 한줄만 나타나서 리뷰하기 용이해집니다
```suggestion
                style = reviewStyle.name,
```